.github/workflows/cron-commit-bump.yml

```diff
@@ -0,0 +1,29 @@
+name: Cron Commit Bump
+
+on:
+  # Allow manual runs
+  workflow_dispatch:
+  # Run every day https://crontab.guru/every-day
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  create-commit-bump:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        # 0 means fetch all commits so we can commit and push in the script below
+        with:
+          fetch-depth: 0
+      - name: Create Commit Bump
+        uses: actions/github-script@v6
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN_PULL_REQUESTS }}
+        # See https://github.com/actions/github-script#run-a-separate-file-with-an-async-function
```

Comment:
Since you script is just invoking git cli directly, you can avoid `actions/github-script` since you don't need JS. Just run directly with something like

```yaml
  run: git config
```